flow-sample-basic/src/main/java/flow/sample/basic/Screens.java

```diff
@@ -0,0 +1,26 @@
+package flow.sample.basic;
+
+import android.content.Context;
+
+final class Screens {
+  private static final String SCREEN = "SERVICE_SCREEN";
+
+  public static <T> T getScreen(Context context) {
+    //noinspection unchecked,ResourceType
+    return (T) context.getSystemService(SCREEN);
+  }
+
+  static final class ContextWrapper extends android.content.ContextWrapper {
```

Comment:
Yeah that seems simpler. Maybe worth revising, will play with it and maybe do a followup.